### ISSUE TITLE
Pods have conflicting source names 

### ISSUE THREAD

**Issue by royherma** [CocoaPods/CocoaPods#5131]
# Report

## What did you do?

Tried installing these two pods:
```
pod 'Google/SignIn'
pod 'MSOutlook-SDK-iOS'
```
## What did you expected to happen?

I expected the project to compile nicely :) 

## What happened instead?

To my delight, those pods both have files named `Core.h` (one is lowercase, but still). I believe i am running into issues that cause a File Not Found for one of the pods because it is looking for a file in the wrong header!

For example, there is a call to `#import "core/NSArray+Extension.h"` in `MSOutlookModels.h`, but I'm getting a `File Not Found`, even though I can clearly see that `NSArray+Extension.h` exists in the project. Digging deeper into my pods I found the `core.h` file name conflict.

I guess I can make the import explicit, prefixing it with its parent folder directory, aka changing:

core.h/NSArray+Extension.h

to

orc/core.h/NSArray+Extension.h

But I don't want to do that bc of two reasons:

1. These pods get updated from time to time, and might introduce new files/overwrite existing changes
2. There are about 350 files I need to do that to.

Any ideas on how to resolve this?

## Podfile

platform :ios, '9.0'
use_frameworks!

target 'Pathlight' do
  pod 'AFNetworking', '~> 2.6', :inhibit_warnings => true
  pod 'Analytics', '3.0.4-alpha', :inhibit_warnings => true
  pod 'CocoaLumberjack', '2.1.0-beta'
  pod 'Crashlytics', '~> 3.3'
  pod 'Fabric', '~> 1.5'
  pod 'Google/SignIn'
  pod 'Heap', '~> 2.2'
  pod 'Intercom', '~> 2.3'
  pod 'JSQMessagesViewController', '~> 7.2'
  pod 'Mixpanel', '~> 2.9', :inhibit_warnings => true
  pod 'Outbound', '~> 0.1'
  pod 'Realm+JSON', '0.2.14'
  pod 'SDWebImage', '~> 3.7', :inhibit_warnings => true
  pod 'SevenSwitch', '~> 1.4'
  pod 'SimpleKeychain', '~> 0.7.0'
  pod 'TSMessages', '~> 0.9', :inhibit_warnings => true
  pod 'XLForm', '~> 3.0', :inhibit_warnings => true
  pod 'SimpleKeychain', '~> 0.7.0'
  pod 'UITextView+Placeholder', '~> 1.1.1'
  pod 'ChameleonFramework', '~> 2.1.0' , :inhibit_warnings => true
  pod 'Segment-Amplitude'
  
  #Outlook
  pod 'MSOutlook-SDK-iOS'

end

**Comment by segiddins** [comment]
This ought to be fixed in CocoaPods 1.0 if you use bracket imports.

**Comment by royherma** [comment]
So tried updating my cocoapods, but it doesn't seem to actually be working (the cocoapods update):

```
roy:k2 royherma$ sudo gem install -n /usr/local/bin cocoapods --pre
Successfully installed cocoapods-1.0.0.beta.6
Parsing documentation for cocoapods-1.0.0.beta.6
1 gem installed
roy:k2 royherma$ pod --version
0.39.0
```

The version is still 0.39.0 - is there something else that i need to do?
